### DISCUSSION TITLE
How can I make Batch List appear even if I only have one file loaded ?

### DISCUSSION THREAD

**Discussion by shodanx2** [mifi/lossless-cut#1278] [General]
And is it possible to drag and drop files directly into the batch list, skipping the open/add dialog

![image](https://user-images.githubusercontent.com/10621885/186375207-9b09a452-45dc-4183-a0ba-21ba973fe49c.png)

(I find this dialog always a bit confusing)
![image](https://user-images.githubusercontent.com/10621885/186374887-149b03f4-e6cc-4d26-9453-4a11576411b2.png)

Being able to just drag and drop files directly into the batch list and have it not ask would be great for the workflow

**Comment by mifi** [comment]
that's a good idea. i've added a feature request #1280
